### PR TITLE
Create a wallet integrator guide

### PR DESCRIPTION
**Pull request by ncjones**
The web3 wallet integrator guide closely resembles the custodial guide but it is using session-based authentication and immersve-conducted KYC.

Ticket Link: https://www.notion.so/immersve/Add-Wallet-Integrator-Guide-a2f621fb466149f5aceec2f294505959

### CODE REVIEW

**Comment by ncjones** [comment]
This depends on #402

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    "accountId": "'${cardholder_account_id}'",
+    "fundingAddress": "'${wallet_address}'",
+    "fundingChannelId": "'${funding_channel_id}'"
+  }' | jq -r '.id')
+```
+
+If the funding address is not related to the user's login wallet address then
+additional "challenge" and "signature" parameters are required to prove
+ownership of the address. In that case the {% link endpoint="create-challenge"
+/%} endpoint would need to be called first.
```
Comment (braden-holgate) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc:176]:
```suggestion
If the funding address is not related to the user's login wallet address then first call the
{% link endpoint="create-challenge" /%} endpoint. The signed challenged can be used 
to prove ownership of the funding address. Include the "challengeId" and "signature" as
additional parameters in the funding source request.
```

**Review by braden-holgate — DISMISSED** [review]

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+The following variables are referenced from the example bash scripts throughout
+this guide.
+
+```bash
+partner_account_id=""<your_partner_account_id>"
+card_program_id="<your_card_program_id>"
+funding_channel_id="<your_funding_channel_id>"
+kyc_redirect_url="<your_app_kyc_redirect_url>"
+imsv_api_host=test.immersve.com
+imsv_client_origin=https://your.domain
```
[the remark this answers is not in the document]
Comment (ncjones) [reply imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc:74]: This is a conscious decision to hint at the format. This is not a domain. It's an origin.

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+  -H "Authorization: Bearer ${imsv_access_token}" \
+  --data '{
+    "accountId": "'${cardholder_account_id}'",
+    "fundingAddress": "'${wallet_address}'",
+    "fundingChannelId": "'${funding_channel_id}'"
+  }' | jq -r '.id')
+```
+
+If the funding address is not related to the user's login wallet address then first call the
+{% link endpoint="create-challenge" /%} endpoint. The signed challenged can be used
```
Comment (catluin) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc:174]:
```suggestion
{% link endpoint="create-challenge" /%} endpoint. The signed challenge can be used 
```

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+For more context and information on card funding and executing deposits and
+withdrawals see the {% link page="guides/card-funding" /%} guide.
+
+
+
+### Complete Cardholder Onboarding
+
+The spending prerequisites endpoint can be used to check whether the KYC and
+requirements for a cardholder have been satisfied.
```
Comment (catluin) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/web3-wallet-card-issuing-integration.mdoc:186]:
```suggestion
The spending prerequisites endpoint can be used to check whether the KYC and other
requirements for a cardholder have been satisfied.
```

**Review by catluin — APPROVED** [review]

**Pull request merged**